docs/conflicts.md

```diff
@@ -47,3 +47,62 @@ The deeper understanding of conflicts has many advantages:
 
 For information about how conflicts are handled in the working copy, see
 [here](working-copy.md#conflicts).
+
+
+## Conflict markers
+
+Conflicts are "materialized" using *conflict markers* in various contexts. For
+example, when you run `jj edit` on a commit with a conflict, it will be
+materialized in the working copy. Conflicts are also materialized when they are
+part of a diff output (e.g. `jj show` on a commit that introduces or resolves a
+conflict). Jujutsu uses a different conflict marker style than Git uses. Here's
+an example:
+
+```
+<<<<<<<
+-------
++++++++
+ apple
+-grape
++grapefruit
+ orange
++++++++
+APPLE
+GRAPE
+ORANGE
+>>>>>>>
+```
+
+That represents a conflict where one side changed "grape" to "grapefruit" and
+the other side has "banana", "grape", and "orange". To resolve the conflict,
+you should keep the side that's preceded by just `+++++++` and apply the diff
+from the side that's preceded by `-------` followed by `+++++++`. In the example
+above, we would presumably do that by replacing "GRAPE" by "GRAPEFRUIT".
+
+For comparison, Git would have presented the above conflict like this:
```

Comment:
I think it would be good to lead with the Git version since people are most likely familiar with it just to make it clear what the actual content of the conflict is in their mind. Then include the explanation that you have to manually figure out the diff from one side and apply it to the other side, and then explain that jj is different and computes one of the diffs for you.

Also, diff3 isn't the default Git conflict marker style, so you might have to explain that explicitly 😂

Reply:
Good idea. Let me know what you think of the rewritten version.